packages/loader/driver-utils/src/network.ts

```diff
@@ -37,9 +37,9 @@ export class GenericNetworkError extends CustomErrorWithProps implements IDriver
     constructor(
         errorMessage: string,
         readonly canRetry: boolean,
-        readonly statusCode: number | undefined,
+        statusCode: number | undefined,
     ) {
-        super(errorMessage);
+        super(errorMessage, { statusCode });
```

Comment:
Isn't it the same as marking property/argument as protected?

Reply:
basically based on the current implementation of  CustomErrorWithProps, but i wanted to used the mechansim provided by the type. I could see the mechansim changing, as member visabilities is a type script only concept for now, in javascript everything is public.